src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -34,13 +34,15 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var childTags = descriptor[nameof(TagHelperDescriptor.AllowedChildTags)].Value<JArray>();
             var documentation = descriptor[nameof(TagHelperDescriptor.Documentation)].Value<string>();
             var tagOutputHint = descriptor[nameof(TagHelperDescriptor.TagOutputHint)].Value<string>();
+            var caseSensitive = descriptor[nameof(TagHelperDescriptor.CaseSensitive)].Value<bool>();
```

Comment:
Would you mind logging an issue in Razor.VSCode and putting it in the next release milestone to update its json converter with these changes?

Reply:
Filed https://github.com/aspnet/Razor.VSCode/issues/377. Could't find any relevant milestones so left if empty.